### PR TITLE
Update Cromwell version

### PR DESCRIPTION
**Pull request by ruchim**
Created & pushed a new docker image of the wdl_runner which uses the Cromwell v26 release.

### CODE REVIEW

**Comment by ruchim** [comment]
I signed it!

File: wdl_runner/workflows/wdl_pipeline.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ inputParameters:
   description: Cloud Storage path for output files
 
 docker:
-  imageName: gcr.io/broad-dsde-outreach/wdl_runner:2017_04_09
+  imageName: gcr.io/broad-dsde-outreach/wdl_runner:2017_04_11
```
Comment (mbookman) [inline wdl_runner/workflows/wdl_pipeline.yaml]: Is the date on this image correct? Has this been baking for 2 weeks?

Comment (ruchim) [inline wdl_runner/workflows/wdl_pipeline.yaml]: Fairly certain the date was when the image was first created--It wasn't pushed until last week.

**Comment by mbookman** [comment]
wdl_runner has been updated to Cromwell 27 with https://github.com/googlegenomics/pipelines-api-examples/pull/66.

**Pull request closed without merging**